### PR TITLE
[release-4.5] Bug 1944254: Update nfd.v4.5.0.clusterserviceversion.yaml

### PR DESCRIPTION
**Pull request by zvonkok**
Bundle build for cluster-nfd-operator is failing on OSBS. It seems to me that OSBS introduced a check and disallows null values in annotations.
cluster-nfd-operator/manifests/olm-catalog/4.5/nfd.v4.5.0.clusterserviceversion.yaml
```
Line 15 in 110fdc7
 containerImage:  
```

is empty. We need to either give it a value to use an empty string "", though I am not sure if the check will allow empty strings or not.

### CODE REVIEW

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Comment by ArangoGutierrez** [comment]
@zvonkok ready for cherry-pick

**Review by wabouhamad — APPROVED** [review]
/lgtm

**Comment by ashcrow** [comment]
(patch manager): "Backport" of https://github.com/openshift/cluster-nfd-operator/pull/149.

**Comment by ashcrow** [comment]
(patch manager): Removing approval due to timing of the merge only. Next patch manager please consider re-approving once the window re-opens.

**Comment by bparees** [comment]
[patch-manager] :rocket: Approved for z-stream by score: 1.80

**Review by wabouhamad — COMMENTED** [review]
/lgtm

**Comment by bparees** [comment]
@zvonkok the nfd upgrade job has not passed since march 29th:

https://prow.ci.openshift.org/job-history/gs/origin-ci-test/pr-logs/directory/pull-ci-openshift-cluster-nfd-operator-release-4.5-e2e-upgrade

someone needs to look into that job or this PR is likely to never merge.

**Comment by bparees** [comment]
@ArangoGutierrez unless you did something to fix the nfd upgrade job, i'm doubtful retesting is going to help.

**Comment by bparees** [comment]
/hold

this job is clearly broken:
https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_cluster-nfd-operator/150/pull-ci-openshift-cluster-nfd-operator-release-4.5-e2e-upgrade/1387877925247782912/build-log.txt

```
[36mINFO[0m[2021-04-29T21:16:00Z] /bin/bash: line 516: python3: command not found 
[36mINFO[0m[2021-04-29T21:16:00Z] {"component":"entrypoint","error":"wrapped process failed: exit status 127","file":"prow/entrypoint/run.go:80","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2021-04-29T21:15:55Z"} 
[36mINFO[0m[2021-04-29T21:16:00Z] error: failed to execute wrapped command: exit status 127 
```

remove the hold once you've fixed the test job.

**Comment by zvonkok** [comment]
@ArangoGutierrez PTAL

**Comment by bparees** [comment]
removing CPA since the test job failure hasn't been resolved yet.

**Comment by sdodson** [comment]
[Patch manager] checking in on this and noting that 4.5 is going EOL shortly, if this isn't receiving attention please close it.

**Comment by ArangoGutierrez** [comment]
4.5 is now deprecated, we now track 4.6, 4.7 and 4.8 
/close

**Pull request closed without merging**
